spec/features/dashboard_spec.rb

```diff
@@ -18,35 +18,61 @@
       "notes"=>"flintstone gummies all dayeee",
       "time_needed"=>5}, @user.id)
     
-      @facade.post({"name"=>"Brush Apollo",
-      "category"=>"chore",
+      @facade.post({"name"=>"crochet",
+      "category"=>"hobby",
       "mandatory"=>"0",
       "event_date"=>"",
       "frequency"=>"weekly",
-      "notes"=>"he shed",
-      "time_needed"=>5}, @user.id)
+      "notes"=>"granny squares",
+      "time_needed"=>60}, @user.id)
 
-      @facade.post({"name"=>"Shower",
-      "category"=>"chore",
+      @facade.post({"name"=>"read a book",
+      "category"=>"rest",
+      "mandatory"=>"0",
+      "event_date"=>"",
+      "frequency"=>"daily",
+      "notes"=>"smut",
+      "time_needed"=>45}, @user.id)
+
+      @facade.post({"name"=>"go on a walk",
+      "category"=>"rest",
       "mandatory"=>"1",
       "event_date"=>"",
       "frequency"=>"daily",
       "notes"=>"",
+      "time_needed"=>30}, @user.id)
+
+      @facade.post({"name"=>"practice juggling",
+      "category"=>"hobby",
+      "mandatory"=>"0",
+      "event_date"=>"",
+      "frequency"=>"monthly",
+      "notes"=>"bowling balls, bowling pins",
       "time_needed"=>15}, @user.id)
 
-      @mandatory_task = @facade.get_tasks(@user.id).first
-      @non_mandatory_task = @facade.get_tasks(@user.id).second
-      @another_mandatory = @facade.get_tasks(@user.id).last
-    end
+      @facade.post({"name"=>"do the dishes",
+      "category"=>"chore",
+      "mandatory"=>"0",
+      "event_date"=>"",
+      "frequency"=>"daily",
+      "notes"=>"",
+      "time_needed"=>15}, @user.id)
 
-    after(:each) do
-      @facade.delete(@mandatory_task.id, @user.id)
-      @facade.delete(@non_mandatory_task.id, @user.id)
-      @facade.delete(@another_mandatory.id, @user.id)
+      @reading = @facade.get_tasks(@user.id).first
+      @walk = @facade.get_tasks(@user.id).second
+      @juggling = @facade.get_tasks(@user.id).third
+      @vitamins = @facade.get_tasks(@user.id).fourth
+      @crochet = @facade.get_tasks(@user.id).fifth
+      @dishes = @facade.get_tasks(@user.id).last
```

Comment:
Note that every time you call that method, it is a separate API call (therefore, 6 different calls making 36 poros total)

```
tasks = @facade.get_tasks(@user.id)

@reading = tasks.first
@walk = tasks.second etc etc etc 
```
Reduces the number of calls you make to 1 :)